File: gmall-publisher/src/main/scala/com/atguigu/gmall/publisher/PublisherApplication.scala
package com.atguigu.gmall.publisher

import org.mybatis.spring.annotation.MapperScan
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
@MapperScan(basePackages = Array[String]("com.atguigu.gmall.publisher.mapper"))
class PublisherApplication {

}

object PublisherApplication {
  def main(args: Array[String]): Unit = {
    SpringApplication.run(classOf[PublisherApplication])
  }
}

File: gmall-publisher/src/main/java/com/atguigu/gmall/publisher/PublisherApplication.java
package com.atguigu.gmall.publisher;

import org.mybatis.spring.annotation.MapperScan;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
@MapperScan("com.atguigu.gmall.publisher.mapper")
public class PublisherApplication {
    public static void main(String[] args) {
        SpringApplication.run(PublisherApplication.class);
    }
}
